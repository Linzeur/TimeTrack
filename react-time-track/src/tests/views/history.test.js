import React from "react";
import { render } from "@testing-library/react";

import History from "../../views/history";

test("Render close projects", () => {
  const { asFragment } = render(<History />);
  expect(asFragment()).toMatchSnapshot();
});
