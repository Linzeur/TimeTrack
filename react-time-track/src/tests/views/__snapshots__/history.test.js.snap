// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render close projects 1`] = `
<DocumentFragment>
  <h1
    class="css-1jtcojf-Subtitle"
  >
    Closed Projects:
  </h1>
  <main>
    <section
      class="css-10oyw64-Section"
    />
  </main>
</DocumentFragment>
`;
